stock-express/src/scripts/populateDataToDB.js

```diff
@@ -13,7 +13,7 @@ const mergeRegionData = async () => {
     // Need to wait for the returned merged data
     const mergedData = await currentData;
     const { data } = await getResponseJSON(
-      `${config.baseurl}/stock/symbol?exchange=${region}`
+      `${config.finnhubBaseurl}/stock/symbol?exchange=${region}`
```

Comment:
kok ngga ada file changenya buat config change? config filenya di gitignore ya? gw msti keep in mind buat ganti di gw punya soalnya

Reply:
config ga gue push soalnya gue masukin secret inget ga, cuman ini doang sih change nya

Reply:
oh secretnya lu directly dri config ya soalnya? i see2